tardis/plasma/base.py

```diff
@@ -258,10 +260,35 @@ def remove_hidden_properties(self, print_graph):
                 print_graph.remove_node(str(item.name))
         return print_graph
 
-class StandardPlasma(BasePlasma):
+    def to_hdf(self, path_or_buf, path='', close_hdf=True, filter=None):
```

Comment:
To add my 2 cents. I think we should have different methods. And I think we need to have only 2 distinctions.
1. case: save all the information required to 'reconstruct' a model. Let's say I run the simulation and save it at the end, I'd be able to continue with the saved information. Store the only uuid of atomdata
2. case: save everything, this includes all intermediately calculated values and the input from the atom database

I don't think we need more filtering. If we need, what would be the usecase?